### ISSUE TITLE
Bad rendering of UI on macOS

### ISSUE THREAD

**Issue by yoonsikp** [mnemosyne-proj/mnemosyne#118]
So I've noticed large differences in UI between my personal build, and the one I downloaded from the website. I'm wondering if this can be remedied? The PR #117  I submitted fixes the Retina rendering at the least, but the blocky buttons are a bit concerning.
### Sourceforge Build
<img width="656" alt="Downloaded from Sourceforge" src="https://user-images.githubusercontent.com/1870062/63573806-f2e7b000-c54b-11e9-8f2f-abe1abf654b2.png">

### Personal Build
<img width="663" alt="Personal Build" src="https://user-images.githubusercontent.com/1870062/63573807-f2e7b000-c54b-11e9-99c2-e205cdefdc00.png">

**Comment by pbienst** [comment]
I'll forward this to our mac developer.

**Comment by devvmh** [comment]
Hey, thanks for submitting. The blocky buttons are probably an artifact of some part of my build process. Are you following the build process in the project README? In particular, I've encoded most of the build into the Makefile. Do you have any suspicions of what's different about your environment or build process that could be yielding the better buttons?

I will try to find time this week to see if I can get some nice looking buttons to show up on my machine too.

**Comment by yoonsikp** [comment]
I ended up using Python 3.7.4 and the following pip3 command

```
pip3 install webob tornado matplotlib numpy sip pillow cheroot pyinstaller PyQtWebEngine
```

Furthermore, I didn't end up using the same brew commands, instead installing the latest `qt`

**Comment by devvmh** [comment]
Update: I tried this today with the latest versions of dependencies and the app doesn't open on my machine. I will keep trying as I have time; it would be nice to get this fixed before the next release. We could also consider bundling the app on your machine if it can make for a better release and you're willing to help

**Comment by devvmh** [comment]
Tried again today with Mnemosyne 2.7 and the latest versions of dependencies (Python 3.5, pyinstaller 4.0-dev, PyQt5 5.13 and Qt 5.13) and I see the correct buttons on my machine. Let's verify this issue is resolved once 2.7 is released.
